daemon/buildinfo.c

```diff
@@ -200,8 +200,67 @@
 
 #define FEAT_YES_NO(x) ((x) ? "YES" : "NO")
 
+// coverity[ +tainted_string_sanitize_content : arg-0 ]
+static inline void coverity_remove_taint(char *s)
+{
+    (void)s;
+}
+
+void read_install_type(void) {
+    char *filepath;
+    filepath = mallocz(sizeof(char) * (strlen(netdata_configured_user_config_dir) + strlen(".install-type") + 2));
```

Comment:
".install-type" should be a `#define` as it is used twice (once to calculate buffer size and second time in sprintf) that would prevent errors in case somebody ever changes it and forgets to change it in other places too

Reply:
In case you will find it useful, we have [a function that reads `.install-type`](https://github.com/netdata/netdata/blob/238a967669288e6d89bc0a2c82bd86520dc99a5c/daemon/analytics.c#L379-L403), could use it as an example if it is good.

Reply:
Hmm, did not see that function. I can’t really use it as-is here, but the code there is probably better than what I’m doing here (which was borrowed from the handling for the `system-info.sh` script).